src/js/components/LoginPage/Login.js

```diff
@@ -1,146 +1,14 @@
-    let errors = {};
-    const { formData } = this.state;
-
-    if (isEmpty(formData.email)) {
-      errors.email = "Email can't be blank";
-    } else if (!isEmail(formData.email)) {
-      errors.email = "Please enter a valid email";
-    }
-
-    if (isEmpty(formData.password)) {
-      errors.password = "Password can't be blank";
-    } else if (isContainWhiteSpace(formData.password)) {
-      errors.password = "Password should not contain white spaces";
-    } else if (!isLength(formData.password, { gte: 6, lte: 16, trim: true })) {
-      errors.password = "Password's length must between 6 to 16";
-    }
-
-    if (isEmpty(errors)) {
-      return true;
-    } else {
-      return errors;
-    }
-  };
-
-  login = e => {
-    e.preventDefault();
-
-    let errors = this.validateLoginForm();
-
-    if (errors === true) {
-      alert("You are successfully signed in...");
-      window.location.reload();
-    } else {
-      this.setState({
-        errors: errors,
-        formSubmitted: true
-      });
-    }
-  };
-
   render() {
-    const { errors, formSubmitted } = this.state;
-
     return (
       <div className="Login">
-        <Row>
-          <Form onSubmit={this.login}>
-            <FormGroup
-              controlId="email"
-              validationState={
-                formSubmitted ? (errors.email ? "error" : "success") : null
-              }
-            >
-              <FormLabel>Email</FormLabel>
-              <Form.Control
-                type="text"
-                name="email"
-                placeholder="Enter your email"
-                onChange={this.handleInputChange}
-              />
-              {errors.email && <Form>{errors.email}</Form>}
-            </FormGroup>
-            <FormGroup
-              controlId="password"
-              validationState={
-                formSubmitted ? (errors.password ? "error" : "success") : null
-              }
-            >
-              <FormLabel>Password</FormLabel>
-              <Form.Control
-                type="password"
-                name="password"
-                placeholder="Enter your password"
-                onChange={this.handleInputChange}
-              />
-              {errors.password && <Form>{errors.password}</Form>}
-
-              <Link to="/resetpassword">Forget Password or UserName?</Link>
-              <Link to="/register"> Not Registered? Sign Up </Link>
-              <div className="d-flex flex-column">
-                <ButtonGroup className="mt-3">
-                  <FormLabel>OR Login with </FormLabel>
-                  <FacebookLoginButton> </FacebookLoginButton>
-                  <GoogleLoginButton> </GoogleLoginButton>
-                  <TwitterLoginButton> </TwitterLoginButton>
-                </ButtonGroup>
-              </div>
-            </FormGroup>
-            <Button variant="primary" type="submit">
-              Sign-In
-            </Button>
-          </Form>
-        </Row>
+        Login
```

Comment:
Surround your text with an element like a paragraph, span, or h6. The div is just for grouping elements on your HTML.

Reply:
Thank you fixed